requirements.dev.txt

```diff
@@ -1,5 +1,5 @@
-cmake==3.13.3
-h3==3.4.3
+cmake==3.18.4.post1
```

Comment:
Maybe we could go with version ```3.18.4```, I don't think it's necessary for us to use this post release. What do you think?

Reply:
I will test this version :smile: